### PR TITLE
Improvements from stream

### PR DESCRIPTION
**Pull request by CarstenKoenig**
that's how I would put it - won't compile because of the type name I don't know (and maybe typos) - but you should be able to figure it out

I basically just changed the `for` loop into a piped `Seq.iter` and removed some braces and `;` you don't need

### CODE REVIEW

File: GithubCommitProcessor/run.fsx [hunk 1]
```diff
@@ -43,35 +43,46 @@ type PushWebHook = JsonProvider<pushSample>
 
 let Run(req: HttpRequestMessage, log: TraceWriter) =
     async { 
-        log.Info(sprintf 
-            "Handling a GitHub WebHook") 
+        log.Info "Handling a GitHub WebHook"
```
Comment (CarstenKoenig) [inline GithubCommitProcessor/run.fsx:46]: you don't format anything - so `sprintf` is not needed

File: GithubCommitProcessor/run.fsx [hunk 2]
```diff
@@ -43,35 +43,46 @@ type PushWebHook = JsonProvider<pushSample>
...
+            }
+          dbContext.Metrics.Add newRecord
+        
+        webhook.Commits 
+        |> Seq.filter (fun c -> c.Committer.Username <> "web-flow") 
+        |> Seq.iter addCommit
           
         let! loaded = dbContext.SaveChangesAsync() |> Async.AwaitTask 
-        loaded |> printfn "Loaded %d records" 
+        printfn "Loaded %d records" loaded
```
Comment (CarstenKoenig) [inline GithubCommitProcessor/run.fsx:84]: IMO the pipe here makes it harder to read - usually I only use a single line `|>` if it reads naturally

File: GithubCommitProcessor/run.fsx [hunk 3]
```diff
@@ -43,35 +43,46 @@ type PushWebHook = JsonProvider<pushSample>
...
+          let newRecord = 
+            {
+                Id = 0
+                CommitId = commitId
+                Repository = repository
+                DateStamp = dateStamp
+                Name = commit.Author.Username
+                NumFilesChanged = commit.Added.Length + commit.Removed.Length + commit.Modified.Length
+            }
+          dbContext.Metrics.Add newRecord
```
Comment (CarstenKoenig) [inline GithubCommitProcessor/run.fsx:77]: you could use a pipe here too - but I think it's overkill

**Comment by csharpfritz** [comment]
Thanks for the suggestions...  I learned a lot from this

**Pull request merged**